Automattic/newspack-plugin
# Revert setting version in release zip file name

### All Submissions:






### Changes proposed in this Pull Request:



Reverts part of changes from #610 – `@semantic-release/github` has to know the file name. This means that in order to have version in the release zip file name, `semantic-release`'s config has to be updated during release, or `semantic-release` should be ran programatically.

### How to test the changes in this Pull Request:

1. Build a release zip
2. Observe it's name is `newspack-plugin.zip`

### Other information:
